forced_photometry/multiband_photometry.ipynb

```diff
@@ -872,14 +872,32 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "df"
+    "#When doing a large run of a large area, save the dataframe with the forced photometry \n",
+    "# so we don't have to do the forced photometry every time\n",
+    "\n",
+    "df.to_pickle('../data/COSMOS_15arcmin.pkl')\n",
```

Comment:
minor organizational comment: I think these result files should be in a subdirectory. Maybe in `../data/results/forced_photometry`, or even just in the same directory as the notebook? I don't the a right answer, just the feeling that we shouldn't mix the input and outputs (especially that the latter is here mostly as a workaround to produce a real demo notebook that doesn't have to run for a long time).